static/js/tab_bar.js

```diff
@@ -1,4 +1,4 @@
-const render_tab_bar = require('../templates/tab_bar.hbs');
+const make_tab_bar_template = require('../templates/tab_bar.hbs');
```

Comment:
We have a standard naming scheme for templates, and so it doesn't make sense to change these names.

I fixed this before merging by renaming the functions to e.g. `render_title_area`, which is a better name anyway.

The UI widget is no longer a "tab bar", so we want to be moving away from that name.  

The whole file could be renamed e.g. `navbar_title.js`